service/src/main/java/uk/gov/hmcts/reform/fpl/enums/UserRole.java

```diff
@@ -0,0 +1,51 @@
+package uk.gov.hmcts.reform.fpl.enums;
+
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.List;
+
+public enum UserRole {
+    DEFAULT {
+        public List<String> getRoles() {
+            return Arrays.asList("caseworker", "caseworker-publiclaw");
+        }
+    },
+
+    LOCAL_AUTHORITY {
+        public List<String> getRoles() {
+            return addRoles(DEFAULT.getRoles(), "caseworker-publiclaw-solicitor");
+        }
+    },
```

Comment:
Why not simply using constructor?
```suggestion
         LOCAL_AUTHORITY("caseworker-publiclaw-solicitor")        
```
And build the list in the constructor?